ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -48,6 +52,15 @@ def _get_group(self, base_url, group_name):
         except Exception, e:
             raise e
 
+    def _get_organization(self, base_url, org_name):
+        url = base_url + self._get_action_api_offset() + '/organization_show?id=' + org_name
+        try:
+            content = self._get_content(url)
+            content_dict = json.loads(content)
+            return content_dict['result']
+        except Exception, e:
```

Comment:
But then again @davidread is right and we shouldn't do it, so let's fix `_get_group` and `_get_organization` to be more explicit :)